packaging/ondemand-bc_access_menu.spec

```
@@ -0,0 +1,65 @@
+# Disable debuginfo as it causes issues with bundled gems that build libraries
+%global debug_package %{nil}
+%global repo_name bc_access_menu
+%global app_name1 bc_access_menu_operations
+%global app_name2 bc_access_menu_allocations
+%global app_name3 bc_access_menu_support
+%global app_name4 bc_access_menu_metrics
+
+%{!?package_version: %define package_version %{major}.%{minor}.%{patch}}
+%{!?package_release: %define package_release 1}
+%{!?git_tag: %define git_tag v%{package_version}}
+%define git_tag_minus_v %(echo %{git_tag} | sed -r 's/^v//')
+
+Name:     ondemand-%{app_name1}
```

```suggestion
Name:     ondemand-%{repo_name}
```

Makes more sense for the RPM name to describe all apps not just the operations app.

---

Added suggestion to batch.

---

> Is that file a copy of the RPM Spec file?

This file : https://github.com/OSC/bc_osc_jupyter/blob/master/.gitlab-ci.yml

It's generic , I'm pretty sure you can copy it from one of the other repos as-is.  You'll need to ensure that code.osu.edu/ondemand org has this project created as mirror of this repo, then push tags to that repo to trigger builds. Used to be automatic before OSU lost the Premium license, that's coming back on different system in future.

---

Not sure what you mean by batch?

---

this was the github term. "Add suggestion to batch" to commit changes.

---

Not sure what that is , since I don't see the change made in the diff.  Normally just click the commit suggestion button on web UI to make the change to pull request.